docs/api.md

```diff
@@ -17,15 +17,15 @@ You can set the filter like this:
 ```
 Expression e = new Parser(reader)
   .withObjectFilter(filter)
-  .parse();
+  .compile();
 ```
 
 The filter can be either a string containing JSLT. The JSLT expression
 needs to return `true` for the object values that should be included.
 So to get the default behaviour you would set the filter to:
 
 ```
-. == null or . == {} or . == []
+. == null or . == {} or . == [] or not(.) == false
```

Comment:
This addition is not correct. As you  can see [here](https://github.com/schibsted/jslt/blob/master/core/src/main/java/com/schibsted/spt/data/jslt/impl/NodeUtils.java#L59) it really is `null`, `{}`, and `[]` that are the omitted values.

The suggested change would, for example, filter out `true`, and `5`, and so on.

Reply:
@larsga Here we have two different implementations:

1. Default implementation: `NodeUtils.isValue(value);` - [here](https://github.com/schibsted/jslt/blob/master/core/src/main/java/com/schibsted/spt/data/jslt/filters/DefaultJsonFilter.java#L30) 
2. Custom implementation: `NodeUtils.isTrue(jslt.apply(value));` - [here](https://github.com/schibsted/jslt/blob/master/core/src/main/java/com/schibsted/spt/data/jslt/filters/JsltJsonFilter.java#L36) 

 `NodeUtils.isValue(value) != NodeUtils.isTrue(value)`

Reply:
Yes, that's true, but the example expression says how to reproduce the default behaviour, which is your number 1.